src/assets/scss/overrides.scss

```diff
@@ -84,3 +75,207 @@
     box-shadow: none !important;
   }
 }
+
+.grey-box-checkbox-button .v-input--selection-controls__ripple {
+  display: none;
+}
+
+.grey-box-checkbox-button .v-input__slot {
+  position: relative;
+  left: 0px;
+  top: 0px;
+}
+
+.designation-buttons button {
+  margin: 0 !important;
+  padding: 0 !important;
+}
+
+.designation-error-col {
+  padding: 5px 10px 0px 10px;
+}
+
+.grey-box-checkbox-button label {
+  font-size: 14px;
+  color: $non-clickable-blue !important;
+}
+
+.v-btn {
+  font-family: inherit !important;
+  font-size: 0.875rem !important;
+  text-transform: none !important;
+  letter-spacing: normal !important;
+  height: 2.5rem !important;
+  box-shadow: 1px 1px rgba(0,0,0,.2) !important;
+  color: white !important;
+  background-color: $app-blue !important;
+  padding: inherit 28px !important;
+}
+
+.v-btn.normal {
+  color: white !important;
+  background-color: $gray6 !important;
+}
+
+.v-label {
+  color: rgba(73, 80, 87, .95) !important;
+}
+
+.sbc-header {
+  .v-btn {
+    background-color: rgba(0, 0, 0, 0)!important;
+    text-transform: none!important;
+    font-weight: normal !important;
+    letter-spacing: normal!important;
+    font-size: .875rem!important;
+  }
+}
+
+.v-btn.primary {
+  color: white !important;
+  background-color: $app-blue !important;
+}
+
+.v-btn.secondary {
+  color: white !important;
+  background-color: $other-bg !important;
+}
+
+.v-btn.outlined {
+  color: $app-blue !important;
+  background-color: white !important;
+}
+
+.v-btn.warning {
+  color: white !important;
+  background-color: $caution !important;
+}
+
+.v-btn.error {
```

Comment:
All these (primary, secondary, outliens, warning, error) will now affect more (all) buttons in theory. We need to make sure things looks the same as before.

Reply:
same as above